packages/ffe-account-selector-react/less/base-selector.less

```diff
@@ -4,25 +4,25 @@
         text-overflow: ellipsis;
         white-space: nowrap;
         overflow: hidden;
-        padding-right: 57px;
+        padding-right: 63px;
         &::-ms-clear {
             display: none;
         }
     }
 
     &__expand-button {
-        height: 23px;
-        width : 23px;
```

Comment:
I'm a bit wary of removing `height` and `width` - could you double-check that the component still looks good in IE?  ([free ready-made VMs from Microsoft with different IE versions can be downloaded here if you need them](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/))